schema.ddl

```diff
@@ -73,3 +144,92 @@ ON subnet (provider_id);
 
 CREATE UNIQUE INDEX IF NOT EXISTS idx_subnet_id
 ON subnet (id);
+
+/*
+ * Model
+ */
+CREATE TABLE IF NOT EXISTS model (
```

Comment:
I think some of this data will be in the model's database, and some in the controller database. In any case we won't need multiple model records in the model database, so maybe leave this one out for now.

Reply:
I wasn't aware there were the two types